13.go

```diff
@@ -767,6 +818,15 @@ func (hs *clientHandshakeState) processCertsFromServer13(certMsg *certificateMsg
 	return hs.processCertsFromServer(certs)
 }
 
+// TODO: Merge with function above
```

Comment:
it will need to be either static/free function (or however it is called in go) or server/clientHandshakeState needs to inherite from common base